_posts/2015-11-25-MAD4Jekyll.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: MAD4Jekyll
+date: 2015-11-25 00:00:00
+homepage: https://github.com/madforjekyll/madforjekyll.github.io
+download: https://github.com/madforjekyll/madforjekyll.github.io/archive/master.zip
+demo: madforjekyll.github.io 
+author: MADness
+thumbnail: MAD4jekyll.png
+license: Apache
```

Comment:
Hello! Thank You for the contribution. Please change the license to "Apache License Version 2.0". Also please use `.markdown` as extension not `.md`.